repltalk/__init__.py

```diff
@@ -1090,24 +1058,28 @@ async def _resolve_report(self, id):
 		)
 
 	async def _get_all_posts(
-		self, order='new', search_query='', after=None
+		self, order='New', search_query=None, after=None
 	):
 		if after is None:
 			posts = await self.perform_graphql(
-				'posts',
-				Queries.get_all_posts,
-				order=order,
-				searchQuery=search_query
-			)
+			'ReplPostsFeed',
+			Queries.posts_feed,
+			options={
+				"order": order,
```

Comment:
Use single quotes instead of double quotes to be consistent with the rest of the code.